### PR TITLE
User Documentation: Creating Nutrition

### PR DESCRIPTION
**Pull request by yemilad**
**Describe the technical changes contained in this PR**
Be clear about purpose and scope of the change

**Previous behaviour**
Describe the product behaviour before this change, if relevant

**New behaviour**
Describe how product behaviour is altered by this change

**Related issues addressed by this PR**
List issue numbers using the "Fixes #xxx" syntax

**Have the following been addressed?**
- [ ] Have test cases been created for all of the changes?
- [ ] Do all of the test cases pass?
- [ ] Has the testing been done using the default docker-compose environment?
- [ ] Are documentation changes required?
- [ ] Does this change break or alter existing behaviour?

### CODE REVIEW

File: docs/Nutrition1.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Meal Planner Documentation
+
+## Overview
+
+## Step 1
+#Meal plan log in page
+
+- Admin shoud be able to access the log in page using the assigned user name and password after inputting the web address [https://goodmealplan.com/](https://goodmealplan.com/) .
+
+![Log in_page-0001](https://imgur.com/vjoJciS.jpg)
```
Comment (rorybray) [inline docs/Nutrition1.md]: Please checkin images alongside the .md files instead of using 3rd party sites.

File: docs/Nutrition1.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Meal Planner Documentation
+
+## Overview
+
+## Step 1
```
Comment (rorybray) [inline docs/Nutrition1.md]: Why are we logging in here if the task is in the admin ui?

**Review by rorybray — CHANGES_REQUESTED** [review]
Just need a couple of changes

File: docs/Nutrition1.md [hunk 3]
```diff
@@ -2,57 +2,54 @@
 
 ## Overview
 
-## Step 1
-#Meal plan log in page
+## Step 1 
+- Access the Admin UI with permissions (Username & Password) assigned as a meal designer.
 
-- Admin shoud be able to access the log in page using the assigned user name and password after inputting the web address [https://goodmealplan.com/](https://goodmealplan.com/) .
+![ADMIN UI login](https://github.com/yemilad/mealplanner/assets/165817613/7552603c-6a27-4917-bb0b-07627941d69e)
```
Comment (rorybray) [inline docs/Nutrition1.md]: Sorry, one more change.  Instead of the full URL to the image, just use the image name.  e.g:

ADMIN UI Login

Comment (rorybray) [reply docs/Nutrition1.md]: And check in the images into the docs/ directory alongside the .md file(s)

**Review by rorybray — CHANGES_REQUESTED** [review]
Just a couple more tweaks to the images.

File: docs/Nutrition1.md (hunk 2 above)
Comment (yemilad) [reply docs/Nutrition1.md]: This has been corrected

File: docs/Nutrition1.md (hunk 1 above)
Comment (yemilad) [reply docs/Nutrition1.md]: Corrections effected.

File: docs/Nutrition1.md (hunk 3 above)
Comment (yemilad) [reply docs/Nutrition1.md]: The corrections have been made.

**Comment by yemilad** [comment]
All the areas of concern have been addressed.

**Review by rorybray — APPROVED** [review]
Thanks!

**Pull request merged**